package-builder/flox-build.mk

```
@@ -296,13 +328,16 @@ define BUILD_template =
 	  exit 1; \
 	fi
 
+  # Create a target for cleaning up the result symlink.
+  $(eval $(call CLEAN_result_link_template,$(_result)))
+
   # Create a helper target for referring to the package by its name rather
   # than the [real] result symlink we're looking to create.
-  .PHONY: $(_pname)
-  $(_pname): $(_result)
+  .PHONY: build/$(_pname)
+  build/$(_pname): $(_result)
 
   # Accumulate a list of known build targets for the "all" target.
```

(non-blocking, nit) To match the other renames.
```suggestion
  # Accumulate a list of known build targets for the "build" target.
```